docs/_articles/en/faq.md

```diff
@@ -57,6 +57,11 @@ A: You must:
 - Install the [Salesforce CLI](https://developer.salesforce.com/tools/sfdxcli) on your computer.
 - Use Java v1.8 or later.
 
+#### Q: How do I update the Salesforce CLI Scanner?
+A: You must:
```

Comment:
@ranekere-sfdc The `You must:` seems out of place, was this checked in by accident?

Reply:
Yes, this was a bad copy-paste. Could you pls fix it as part of this PR?